aws-codeartifact-domain/src/test/java/software/amazon/codeartifact/domain/ReadHandlerTest.java

```diff
@@ -97,6 +97,8 @@ public void handleRequest_simpleSuccess() {
         final ResourceModel desiredOutputModel = ResourceModel.builder()
             .domainName(DOMAIN_NAME)
             .domainOwner(DOMAIN_OWNER)
+            .name(DOMAIN_NAME)
+            .owner(DOMAIN_OWNER)
```

Comment:
Do we need an explanation here of why we're storing the same information twice? (domainOwner + owner, and domainName + name)

Reply:
There is an explanation in the model json itself. It says that those new fields are used in GetAtt. Not sure if we need to explain more than that

Reply:
Thanks

Reply:
I did some testing and we may not need to have both for reading. We need them both to be present in the model but we don't need both unless we want both `DomainName` and `Name` to be available to `GetAtt`